memory/libhugetlbfs.py

```diff
@@ -83,11 +83,11 @@ def setUp(self):
                                                       verbose=False,
                                                       shell=True)
             if 'HugePages_' not in Hugepages_support:
-                self.error("No Hugepages Configured")
+                self.cancle("No Hugepages Configured")
             memory.set_num_huge_pages(pages_requested)
             pages_available = memory.get_num_huge_pages()
         else:
-            self.error("Kernel does not support hugepages")
+            self.cancle("Kernel does not support hugepages")
```

Comment:
Here too.

Reply:
no i think we need to come out from test if test is not supported so `cancle` will be good option

Reply:
i meant the typo :)